src/Slot-Examples/BooleanSlot.class.st

```diff
@@ -106,7 +87,7 @@ BooleanSlot >> todo [
 	'
     - adding a Booleanslot in between --> offsets need to be updated?
 	 - removing BooleanSlot in the middle	
-	 - adding booleanslot higher in the hierarchy: move baseselot (same as in TODO for DictionarySlot)
+	 - adding booleanslot higher in the hierarchy: move baseselot (same as in TODO for PropertySlot)
```

Comment:
Having TODOs as methods is no so nice, at least not if they are only understandable by you...

If they are to stay here I'd clarify it a bit more.
For example, it's not clear wether booleanslot = Booleanslot or BooleanSlot. Same with baseslot...

Reply:
removed the TODO, will open issue tracker entry (I have now a plan for how to fix it...)